src/ui/src/pages/auth/signup.tsx

```diff
@@ -64,8 +65,7 @@ export const SignupPage = React.memo(() => {
             toggleURL={`/auth/login${window.location.search}`}
             title='Get Started'
             // Need to encapsulate so that newline is properly escaped.
```

Comment:
**Nit**: This comment should move to where the string is defined, rather than where it's used.

Reply:
Good call 👍